packages/rsc-router/package.json

```diff
@@ -8,6 +8,7 @@
   ],
   "exports": {
     "./entry-server": "./src/entry-server.ts",
+    "./streams": "./src/server/streams.tsx",
```

Comment:
I'm using the stream helpers in the server entry. It might be better to move these into a shared package?

Reply:
Sure, we just didn't organize well yet :)